CollAction/Controllers/ProjectsController.cs

```diff
@@ -163,7 +163,7 @@ public ViewResult StartInfo()
                 "Hi!<br>" +
                 "<br>" +
                 "Thanks for submitting a project on www.collaction.org!<br>" +
-                "The CollAction Team will review your project as soon as possible � if it meets all the criteria we�ll publish the project on the website and will let you know, so you can start promoting it! If we have any additional questions or comments, we�ll reach out to you by email.<br>" +
+                "The CollAction Team will review your project as soon as possible � if it meets all the criteria we�ll publish the project on the website and will let you know, so you can start promoting it! If we have any additional questions or comments, we�ll reach out to you by email.<br>" +
```

Comment:
I get that you're having issues with the fonts or certain characters, but now it won't display for me. 

I suggest excluding these changes for now.